vertical-pod-autoscaler/pkg/admission-controller/logic/server.go

```diff
@@ -73,7 +73,6 @@ func (s *AdmissionServer) admit(data []byte) (*v1.AdmissionResponse, metrics_adm
 	var err error
 	resource := metrics_admission.Unknown
 
-	response.UID = ar.Request.UID
```

Comment:
Why remove this?

Reply:
It was defined twice. See few lines above.